src/Auth.php

```diff
@@ -215,8 +215,7 @@ public function check()
         }
 
         // if user is not logged in, then show login form
-        $l = new \atk4\ui\App();
-        $l->title = $this->title;
+        $l = new \atk4\ui\App($this->title);
```

Comment:
you already have `$this->app->title`

Can be better, leave the '$this->title' to be used as the title for the form

And assing $app->title from $this->app->title, to have a much linear implementation.

Sorry, because probably in chat, when we discuss this, i pointed you in a wrong direction.

Reply:
Reverted back to a more basic implementation. Regarding the titles we can discuss about it when we start the localization project.